components/Converters/samples/DoubleToVisibilityConverterSample.xaml

```diff
@@ -0,0 +1,28 @@
+<!--  Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information.  -->
+<Page x:Class="ConvertersExperiment.Samples.DoubleToVisibilityConverterSample"
+      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:converters="using:CommunityToolkit.WinUI.Converters"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:local="using:ConvertersExperiment.Samples"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      mc:Ignorable="d">
+
+    <Page.Resources>
+        <converters:DoubleToObjectConverter x:Key="DoubleToVisibilityConverter"
+                                            FalseValue="Collapsed"
+                                            GreaterThan="5"
+                                            TrueValue="Visible" />
```

Comment:
Again, let's not set the default values within the XAML when they're the defaults.

Since this is similar to the DoubleToObject example (which I love showing the slider with both greater and less), maybe we use the ConverterParameter to invert in this one?

Reply:
Aaaah.. just realized I was using a `DoubleToObjectConverter` vs. a `DoubleToVisibilityConverter`, hence why it crashes when the `FalseValue` or `TrueValue` wasn't set.

Oops - copy+paste error :(. Fixed now!